internal/project/project.go

```diff
@@ -51,7 +52,13 @@ func (s *snapshot) GetLineMap(fileName string) *ls.LineMap {
 	if s.project.getFileVersion(file) == scriptInfo.Version() {
```

Comment:
I still don't really understand why we're doing this at all; it seems to me that we should not cache line maps on script infos at all, instead do it per-source-file, and then just always use that. But I seem to remember someone saying that was wrong for some reason, so, this hack seems better than nothing?

Reply:
I recall a conversation around line maps differing between LS and CLI

Reply:
That's a different thing but yeah they do.